src/coreclr-debug/activate.ts

```diff
@@ -40,12 +40,14 @@ async function checkForInvalidArchitecture(platformInformation: PlatformInformat
             eventStream.post(new DebuggerPrerequisiteFailure("[ERROR] The debugger cannot be installed. The debugger requires macOS 10.12 (Sierra) or newer."));
             return true;
         }
-        else if (platformInformation.architecture !== "x86_64") {
-            if (platformInformation.isWindows() && platformInformation.architecture === "x86") {
-                eventStream.post(new DebuggerPrerequisiteWarning(`[WARNING]: x86 Windows is not currently supported by the .NET Core debugger. Debugging will not be available.`));
-            } else {
-                eventStream.post(new DebuggerPrerequisiteWarning(`[WARNING]: Processor architecture '${platformInformation.architecture}' is not currently supported by the .NET Core debugger. Debugging will not be available.`));
-            }
+        else if (platformInformation.isWindows() && platformInformation.architecture === "x86") {
```

Comment:
>platformInformation.architecture === "x86" [](start = 52, length = 42)

Instead how about `!== "x86_64"` and change the 'x86' in the error message to be replaced with `${platformInformation.architecture}`.